Model-ID-Reference.mediawiki

```diff
@@ -1,5143 +0,0 @@
-
-{| class="wikitable sortable zebra"
```

Comment:
Ah that's what it was I couldn't remember! Gollum didn't support a sortable type. Note to self: nag github for that feature.